components/access-list/ui/AccessListMember.js

```diff
@@ -0,0 +1,78 @@
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { Body1 } from '../../ui/AltTypography'
+import useTruncatedAddress from '../../hooks/useTruncatedAddress'
+import useEns from '../../hooks/useEns'
+
+const AccessListMember = ({ size, address, src }) => {
+  const { ens } = useEns(address)
+  const { truncatedAddress } = useTruncatedAddress(address)
+
+  return (
+    <Link href={`/token/${address}`} passHref>
+      <Wrapper>
+        <Avatar src={src} size={size} />
+        {size !== 'small' && (
+          <Address color="vibrantPixel" size={size}>
+            {ens || truncatedAddress}
+          </Address>
+        )}
+      </Wrapper>
+    </Link>
+  )
+}
+
+AccessListMember.propTypes = {
+  size: PropTypes.oneOf(['large', 'medium', 'small']).isRequired,
+  address: PropTypes.string.isRequired,
+  src: PropTypes.string.isRequired,
+}
+
+const Wrapper = styled.div`
+  cursor: ${props => props.theme.cursors.select};
+`
+
+const SIZES = {
+  large: 15,
+  medium: 6.4,
+  small: 2.4,
+}
+
+const Address = styled(Body1)`
+  text-shadow: 0px 2px 4px #000000;
+  padding-top: 12px;
```

Comment:
just checking, were we going to use `rem` as a standard?

Reply:
Good catch, changing!